lib/CL/devices/pthread/pthread.c

```diff
@@ -117,6 +113,10 @@ struct data {
   mem_regions_management* mem_regions;
 #endif
 
+  /* Lock for command list related operations */
+  pthread_mutex_t cq_lock __attribute__ ((aligned (CACHELINE_SIZE)));
```

Comment:
OK, i have added a bit of CMake code to detect cacheline size on linux, with fallback to 64. It's also possible to define at CMake run.